pkg/apis/policy/install/install.go

```diff
@@ -34,5 +35,6 @@ func init() {
 func Install(scheme *runtime.Scheme) {
 	utilruntime.Must(policy.AddToScheme(scheme))
 	utilruntime.Must(v1beta1.AddToScheme(scheme))
-	utilruntime.Must(scheme.SetVersionPriority(v1beta1.SchemeGroupVersion))
+	utilruntime.Must(v1.AddToScheme(scheme))
+	utilruntime.Must(scheme.SetVersionPriority(v1beta1.SchemeGroupVersion, v1.SchemeGroupVersion))
```

Comment:
v1 should always be before v1beta1

Reply:
So it seems like we should prefer v1beta1 now and then switch in the next release. Ref: https://github.com/kubernetes/kubernetes/pull/95718#discussion_r520969477

Reply:
Fair, we'll switch in the next release. Add a todo here.

Reply:
Added a TODO here.